src/app/src/components/UserCookiePreferences.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+
+import Divider from '@material-ui/core/Divider';
+
+import COLOURS from '../util/COLOURS';
+import Button from './Button';
+
+import {
+    userHasAcceptedGATracking,
+    acceptGATrackingAndStartTracking,
+    rejectGATracking,
+    clearGATrackingDecision,
+} from '../util/util.ga';
+
+const componentStyles = Object.freeze({
+    header: Object.freeze({
+        padding: '0 1rem',
+    }),
+    content: Object.freeze({
+        padding: '2rem 1rem',
+        display: 'flex',
+        flexDirection: 'column',
+    }),
+    buttons: Object.freeze({
+        padding: '2rem 0',
+        display: 'flex',
+        justifyContent: 'flex-end',
+    }),
+});
+
+class UserCookiePreferences extends Component {
+    state = {
+        accepted: userHasAcceptedGATracking(),
+    }
+
+    acceptGDPRAlert = () => this.setState(
+        state => Object.assign({}, state, { accepted: true }),
+        () => {
+            clearGATrackingDecision();
+            acceptGATrackingAndStartTracking();
+        },
+    );
+
+    rejectGDPRAlert = () => this.setState(
+        state => Object.assign({}, state, { accepted: false }),
+        () => {
+            clearGATrackingDecision();
+            rejectGATracking();
+        },
+    );
+
+    render() {
+        return (
+            <div className="margin-bottom">
+                <Divider />
+                <h3 style={componentStyles.header}>
+                    Cookie Preferences
+                </h3>
+                <div style={componentStyles.content}>
+                    <div>
+                        The Open Apparel Registry uses cookies to collect and analyze
+                        site performance and usage. By clicking the Accept button, you
+                        agree to allow us to place cookies and share information with
+                        Google Analytics. For more information, please visit our{' '}
+                        <a
+                            href="https://info.openapparel.org/tos/"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            Terms and Conditions of Use and Privacy Policy.
+                        </a>
+                    </div>
```

Comment:
Consider extracting this tex into a component that can be used both here and in `GDPRNotification.jsx`. If we need to update this language in the future it will eliminate the possibility of accidentally updating it in one place and not another.

Reply:
Good call! Thank you for the review!